packages/components/src/menu/stories/index.story.tsx

```diff
@@ -64,300 +64,337 @@ const meta: Meta< typeof Menu > = {
 			source: { excludeDecorators: true },
 		},
 	},
-};
+} satisfies Meta< typeof Menu >;
```

Comment:
Is there a good reason for this change?

When I run the `csf-2-to-3` codemod it keeps that type above, as it was in CSF2.

Try it out on `trunk`:

```
npx storybook@8.4.7 migrate csf-2-to-3 --glob="packages/components/src/menu/stories/index.story.tsx"
```

Reply:
Also curious. I'd prefer to keep doing it the old way for consistency, if there are no benefits. (I noticed someone recently used `satisfies` for the `Badge` stories.)

Reply:
> (I noticed someone recently used satisfies for the Badge stories.)

That's where I got it from, I thought it was a pattern we were onboard with.

Ran the migration tool and reverted it to how it was before.